radicle-cli/examples/rad-remote.md

```diff
@@ -21,12 +8,13 @@ $ rad remote add did:key:z6Mkt67GdsW7715MEfRuP4pSZxJRJh6kj6Y48WRqVv4N1tRk --alia
 Now, we can see that there is a new new remote in the list of remotes
 
 ```
-$ rad remote list
+$ rad remote
 bob
 rad
-z6MknSLrJoTcukLrE435hVNQT4JUhbvWLX4kUzqkEStBU8Vi
 ```
 
+You can see both bob and rad as remotes.  The remote `rad` is our remote of the project.
+
```

Comment:
why we do not keep both to show that we support `rad remote` as a shortcut to `rad remote list`?

Reply:
I reverted it back to use `rad remote list`.  The update was an accident, while trying to get `rad remote -v` working